src/main/java/com/extendedclip/papi/expansion/player/PlayerUtil.java

```diff
@@ -136,4 +136,31 @@ public static int getEmptySlots(Player p) {
         }
         return slots;
     }
+    
+    public static int getExperienceAtLevel(Player player) {
+        return getExperienceAtLevel(player.getLevel());
+    }
+
+    private static int getExperienceAtLevel(int level) {
+        if (level <= 15) {
+            return (level << 1) + 7;
+        }
+        if (level >= 16 && level <= 30) {
+            return (level * 5) - 38;
+        }
+        return (level * 9) - 158;
+    }
+
+    public static int getTotalExperience(Player player) {
+        int experience = Math.round(getExperienceAtLevel(player) * player.getExp());
+        int currentLevel = player.getLevel();
+        while (currentLevel > 0) {
+            currentLevel--;
+            experience += getExperienceAtLevel(currentLevel);
+        }
+        if (experience < 0) {
+            experience = Integer.MAX_VALUE;
+        }
```

Comment:
if the current experience is less than 0 (which is most likely impossible to happen afaik) why setting it to the maximum value of an integer (2,147,483,647) instead of 0?